config/sitemap.rb

```diff
@@ -0,0 +1,39 @@
+# Set the host name for URL creation
+SitemapGenerator::Sitemap.default_host = "https://coderdojo.jp"
+
+SitemapGenerator::Sitemap.create do
+  # Put links creation logic here.
+  #
+  # The root path '/' and sitemap index file are added automatically for you.
+  # Links are added to the Sitemap in the order they are specified.
+  #
+  # Usage: add(path, options={})
+  #        (default options are used if you don't specify)
+  #
+  # Defaults: :priority => 0.5, :changefreq => 'weekly',
+  #           :lastmod => Time.now, :host => default_host
+
+  add root_path,        priority: 0.9
```

Comment:
This line is not necessary so removed by this commit https://github.com/coderdojo-japan/coderdojo.jp/commit/35d5ae1ca171fda1c3df23bc48b1dd5d0e3863bf.